Add note about normalizing usernames if it did not match K8S naming format (#1651)
codenvy/che-docs: modules/installation-guide/partials/proc_configuring-namespace-strategies.adoc

allowUserDefinedWorkspaceNamespaces
----

> == Handling incompatible usernames or user ID-s

Reviewer: ```suggestion
== Handling incompatible usernames or user IDs
```

> {prod-short} server automatically checks usernames and\or ID-s for compatibility with Kubernetes objects naming convention before creating a {orch-namespace} from a template.

Reviewer: ```suggestion
{prod-short} server automatically checks usernames and IDs for compatibility with Kubernetes objects naming convention before creating a {orch-namespace} from a template.
```

> In case of incompatible user name or ID, it will be reduced to nearest valid name by replacing groups of unsuitable symbols with an `-` symbol. To avoid collisions,

Reviewer: ```suggestion
Incompatible username or IDs are reduced to the nearest valid name by replacing groups of unsuitable symbols with the `-` symbol. To avoid collisions,
```

> some random 6 symbols suffix is added and result is stored into preferences for re-use.

Reviewer: ```suggestion
a random 6-symbol suffix is added and the result is stored in preferences for reuse.
```
